custom/icds_reports/urls.py

```diff
@@ -191,6 +191,7 @@
     url(r'^governance_apis/state_names/', GovernanceStateListAPI.as_view(), name='governance_apis_state_names'),
     url(r'^governance_apis/cbe/', GovernanceCBEAPI.as_view(), name='governance_apis_cbe'),
     url(r'^governance_apis/vhsnd/', GovernanceVHNDSAPI.as_view(), name='governance_apis_vhnds'),
+    url(r'^household_members_data/',DemographicsAPI.as_view(), name='household_members_data')
```

Comment:
Not sure if this is committed yet, but it might be worth adding prepending something for all the bihar APIs at the front like we do with governance ones. so it would be `bihar_demographics/household_member_data/` or similar so that all the bihar ones share a path and if we have other household ones it is clear what we mean.

Reply:
I can check with Nitish.